ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -54,8 +54,8 @@ object Reporter {
           s.println()
           s.println(cyan(suite))
 
-          for (LedgerTestSummary(_, test, _, result) <- summaries) {
-            s.print(cyan(s"- $test ... "))
+          for (LedgerTestSummary(_, name, desc, _, result) <- summaries) {
+            s.print(cyan(s"- [$name] $desc ... "))
```

Comment:
```suggestion
          for (LedgerTestSummary(_, name, description, _, result) <- summaries) {
            s.print(cyan(s"- [$name] $description ... "))
```
I would recommend spelling out names unless there is a compelling reason not to do so.